Automattic/newspack-blocks
# fix: hide reCAPTCHA badge everywhere, not just in modal checkout

### All Submissions:






### Changes proposed in this Pull Request:



Migrates the CSS that hides the reCAPTCHA badge to the main Plugin, so that the badge is hidden everywhere (not just in the modal checkout). Requires https://github.com/Automattic/newspack-plugin/pull/3234

### How to test the changes in this Pull Request:

1. On `epic/ras-acc` for both this repo and the main Plugin, observe that the reCAPTCHA badge is shown at the lower-right corner of most pages:

<img width="89" alt="Screenshot 2024-07-09 at 4 10 22 PM" src="https://github.com/Automattic/newspack-blocks/assets/2230142/99a6c67a-7af3-4a50-8a72-436a6f7a90e8">

2. Check out this branch and https://github.com/Automattic/newspack-plugin/pull/3234 and confirm that it's hidden everywhere now.

### Other information:
